docs/guides/ha-dr.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Understanding high availability and disaster recovery for Qiskit Runtime
+description: Understanding high availability and disaster recovery for Qiskit Runtime
+
+---
+
+# Understanding high availability and disaster recovery for Qiskit Runtime
+
+High availability (HA) is the ability for a service to remain operational and accessible in the presence of unexpected failures. Disaster recovery is the process of recovering the service instance to a working state.
+
+Qiskit Runtime is a regional service that fulfills the defined [Service Level Objectives (SLO)](/docs/resiliency?topic=resiliency-slo) with the Standard plan. For more information, see [Service Level Agreement (SLA)](https://www.ibm.com/support/customer/csol/terms/?id=i126-9268&lc=en). For more information about the available IBM Cloud regions and data centers for Qiskit Runtime, see [Service and infrastructure availability by location](/docs/overview?topic=overview-services_region).
+
+The quantum computing service is deployed using a Multi Zone Regional deployment model.
+
+
+It available in the following regions:
+
+- Washington DC (us-east)
+- Frankfurt (eu-de)
```

Comment:
I think this statement should be moved under the High availability architecture header